### PR TITLE
Update the fetch wallet name in docs

### PR DESCRIPTION
**Pull request by VijaySharma815**
- Update the fetch wallet name in through the documentation.

### CODE REVIEW

File: docs/basics/convert-native-fet.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
     **Do not share this with anyone! If you lose this, you cannot recover your assets.**
 
-     <img src="/images/native_and_erc20/keplr_wallet.png" alt="Keplr Wallet" class="center" style="display: block; margin-left: auto; margin-right: auto;width:400px;">
+     <img src="/images/native_and_erc20/keplr_wallet.png" alt="Fetch Wallet" class="center" style="display: block; margin-left: auto; margin-right: auto;width:400px;">
 
-2. Send the amount of tokens you wish to transfer from your cosmostation mobile wallet or your ledger to the generated address on your fetch browser wallet.
+2. Send the amount of tokens you wish to transfer from your cosmostation mobile wallet or your ledger to the generated address on your fetch wallet.
 3. Once you have done so, browse to [https://token-bridge.fetch.ai/](https://token-bridge.fetch.ai/)
 4. Switch to the **Native to ERC20** tab.
 5. Click on **connect browser wallet** and allow the token bridge to connect to your browser wallet.
```
Comment (VijaySharma815) [inline docs/basics/convert-native-fet.md:14]: token bride

- connect browser wallet ->  It should be connect fetch wallet 
- Unable to detect the browser extension -> It should be unable to detect the browser extension. 

@5A11

Comment (5A11) [reply docs/basics/convert-native-fet.md:14]: Open issues on the token bridge repo for the above

**Review by 5A11 — APPROVED** [review]
lgtm

File: docs/basics/convert-native-fet.md (hunk 1 above)
Comment (5A11) [reply docs/basics/convert-native-fet.md:14]: @VijaySharma815

Comment (VijaySharma815) [reply docs/basics/convert-native-fet.md:14]: @5A11 could you share the repo url with me?

**Pull request merged**